packages/components/src/tabs/styles.ts

```diff
@@ -163,6 +163,7 @@ export const Tab = styled( Ariakit.Tab )`
 		cursor: pointer;
 		line-height: 1.2; // Characters in some languages (e.g. Japanese) may have a native higher line-height.
 		font-weight: 400;
+		font-size: inherit;
```

Comment:
Should this be `$font-size-medium` instead?

cc @WordPress/gutenberg-design

Reply:
I don't mind, but just noting that for now, I used inherit to avoid any behavior change in WordPress.

Reply:
> I don't mind, but just noting that for now, I used inherit to avoid any behavior change in WordPress.

My understanding is that wouldn't be a behavior change, it'd just be making explicit what we're already expecting as a style in WordPress.

(It's also what's specified in [the Figma designs](https://www.figma.com/design/804HN2REV2iap2ytjRQ055/WordPress-Design-System?node-id=991-34616&p=f&t=Pb2MWSs4UgiphcFf-0))

Reply:
"Medium" is 13px, which is almost certainly what it already inherits, and is the right thing here.

I do think there's a chance we can revisit the typography scale in the design system file, to be clear. But no concrete plans.